intervals_2.html

```diff
@@ -918,12 +985,25 @@
     }
 
     function saveAsPdf() {
-      html2canvas(document.body, {
+      const pdfResultsContainer = document.createElement('div');
+      const container = document.getElementById('container');
+      const resultsContainer = document.getElementById('results-container');
+      container.appendChild(pdfResultsContainer);
+      pdfResultsContainer.appendChild(resultsContainer);
+      pdfResultsContainer.classList.add('pdf-results-container');
+
+      const generatePdfButton = document.querySelector('.generate-pdf-button');
+      generatePdfButton.classList.add('remove-button');
+      setTimeout( () => {
```

Comment:
instead of removing the button let's put the button outside the element that we create the pdf out of

Reply:
Done.